Source/Fuse.Reactive.JavaScript/Tests/UXExpressionsTest.uno

```diff
@@ -82,37 +82,6 @@ namespace Fuse.Reactive.Test
 				root.StepFrameJS();
 
 				Assert.AreEqual("Hello foo : FOO!", e.t1.Value);
-
-				if defined(FUSELIBS_NO_TOASTS)
```

Comment:
Why has this bit been removed? Shouldn't this always run now since the define would be true?

Reply:
@mortoray: Read the commit message. This test with this define enabled has been broken since before 1.0. Since https://github.com/fusetools/fuselibs/commit/d2aa412e0bbb77bc961ffc79a98c9cd3787dabc7, to be accurate.